ethtest/state_test_context.go

```diff
@@ -79,5 +87,9 @@ func (s *stateTestContext) GetResult() txcontext.Result {
 }
 
 func (s *stateTestContext) String() string {
-	return fmt.Sprintf("Test path: %v\nDescription: %v\nFork: %v\nPost number: %v", s.path, s.description, s.fork, s.postNumber)
+	return fmt.Sprintf(
+		"Test path: %v\n"+
+			"Test label: %v\n"+
+			"Fork: %v\n"+
+			"PostNumber: %v\n", s.path, s.testLabel, s.fork, s.postNumber)
```

Comment:
nit: since in the JSON files the hierarchy is Post > Fork, these should be printed in the reverse order.

Reply:
good point